anw/TD-ANWv2024.1.md

```diff
@@ -0,0 +1,746 @@
iTGKDQ0TJU9Q2ufKaTnX9hP+kkS9xUUFCbmpv21bcUYZimz2bGjTV1PUUrKE7Qx7Q1aviqbQvoOkDa5f67bztcmMZ046TExa/eYjNYz68uog78fFai5cx02PnwtqThHbdtocfT09FRm7pwywrOoQhm5+yY9TilTnlBm9CM5KeR4vKT6nzptCe0qPEM33tBF1spDmekStXazf52q1vm0lOO5r1iTRzff3J0KSrU+q1XZ3sERUmfm7BVUVPY9jRw9RV3PqfxWYqA93N2lbZvNnQ6/TVLGy8tLxUszis7PGIwWAFYDowWARTRntHSjwpvuth0nxKSwSZqcsljSeLMPDomT05nH52+Q05teQf0lnY3WsOTJykx8Q6vWFsjpiP6ajNvo0KEtzXlkPW1SGz/n8ebdpUs3MVrTf/mkbMCllRekz+joUXKqxeNp3dpGVcoI3Xv/XAoLj6Ow/rHk7eUmRqP9tb5SZu68Z5xeMbIJ4dO2DS/uF0PwQMo86t27LwUEtBej1bNnd5nTgkWbGoxTSYVmxPg0j+MQEhxFB1RZfz836h8Wq/ofSB0DbpI87ofn7eXVmtL+9hYNiIqjt97lU7rT5OPjK7HhMQYHh9BTK7PqTwJraezYmTL3rsr09A+LoXA1p6FDH6B77nE0WuvWl1NGtlaHzQyncTsRkUnk72+TmG7ZdpQSEscrY9eO0rd+SGMmzNbmr+bFRqu0sk6NOUnGyXMLDPSRud/SrZvEMTwiXpnjScQGVDehm9OP1MdRWyM2Wlx/xszlFBMTR5GR8RSmxu3X3kPSg26LkH4WLX5ZDBeveevWXhIj8/MFowXA1QFGCwCLaM5o8cmOn19HyWOjkJg4rsFo8cbLJ0S3dLtVTi3iE5KptPw8de8RSofeJKpWdUeMmEoFJWdpzdp8Ka+3y21EDrhLTnzaKBPCp0R8WsJGi/uc9tBS6TMz67jkcfnAwEAxKh4eblLmqZU5Yqa4jec2vipp7du3k3pz561rMFraZl5H0TF3UWzcWDELi5e8rMb1HZVVnKd2vm5iNoJu6+FktNhA+vr6UHbuaZl/u3Zt5LRKviXbWyuGbXvmMemH625MO0jrn6uSsfCp2yOPrpETLS8v7cSNTwDj40fSshWZNG9+mrQZEqy9jnRz85J7nuvO/C+cjdaz5cqc/Uv68vBwl3kMS54uY+wTEiGxcXPTDNjipemUvv0ojVNGi+/1OPAajBw1XV7zVVTXyjw4vl6tPdUafidlcupf8bGio4fQggVpEuOhwyaIeTYarZWr81QMzypdoBc31cjY2WhxLH6/JF1O/bgdm81bjF1jzxiMFgDWA6MFgEU0Z7Q4LSvnlGzIrIW/T5NNlF8z8T1v9pNT5srfv1u4WcrzyQXfb93+MQ0ecj/tKvqKli3PFEOgt8sGpUfPXtKWvLry0E5DOlx3HbHJmTBhhrTFGze3ZbN5qr43S3k2HPrrtojIIXLi9eyfq8XcsNHgjfvhGU/RTsM3QXwtViawW7fbZcPXX0Vef30nVaeVfIfUsaN26jLtoSViHPR6Ygbb+Er58PBEMSb6R+DyOq3+1SePzWazySkW3+tjZ+Pi49OW/P07kLt7vUlcnS0nRZyvxYa/efqmoc0q1ebdymgZzenq1N3yrReXbdXKTWLYu3eUxCZx0Di5nzlrhYqljVat3kV/33KEkoc/2DAP/aqP3dvbmwICO4l5yi8+a++7/tsqPf7jxk+WGKU+XVh/6thZ5jx+wiwZX5++USrfjZ5/8TUp37NXP4njwkWbaVf+58Tfyw0eMlGZ3NEydvMzBqMFgPXAaAFgEc0ZLV36CYdxs7aXd/z3DMY8e1lj+cbbbEgz5Bll/HWgc55jm5oc+9TvG1NjdRtvs+myxnzjfW7eZ8rUaK8O9fLLV2VResaHpvbM7ZrH0Vi+XeZ+zevgnG+vZ27b2Kf5V5nGNhuvY0w339vL6YLRAsB6YLQAsIiWGi3ox0v/Ns0YX/5eyvhvLP6fBaMFgPXAaAFgETBaVsj5dRlkF4wWANYDowWARcBoQVdbMFoAWA+MFgAWwUbrvpRPKHXdl/LLtwNvQJC14h8XsNF6eA6MFgBWAaMFgEVcVk7rgWnHKWX6B5QweA8lDqmiBCW+sgYNdVbSL+y6y6hhdg1OtsuY3pyMbf8UmcfsMH6TzHP6OaqpeTe1ri2Ree10Oa27aRy69GeMn7tfzTlOS5Z9Zn48AQAuAkYLAIvgE63a2su077WvqKLyBGVlf0w5ucco7a8fiNasfY9Sn36PFv/hHVq45B1asPhtmvnIYZr1mzdpttLEKYdo0oOvi4aN3k/JY2poxNgaioyvpgEJeykqfi/FDNpLsUmaopOqKXZQFcUlVWnXQXvU3/VSf8ckVFJMYr0SKihqYDndGV8hihpYQREx5Y6KLqO+EWUUGlluUBn16FPqpF6hpRQSXtao+keVX1GRMRUUGfsDFKON+SdpYCPtNiUVi/A7ncfN6qdkni/LGJuewaUSN44fxzNcxVWLbzkNUFeOPYvXRFubSm3tlGJlDatknVmRat0jBlZT8ugaGnN3jTwbM+Ycludm0dJ3aHUqP1Pv0kub3qfsnGNUXnGC9u372v5QAgBcDowWAFcJPuH6saqr++Fik3clXbrEqruiLl501oULzrp4UbXJfZql+vm5I/E0zds5PnaZY6yrtlaXfY3Ma+qgRp4Ts9RTZx4uAMDFwGgBAAAAALgIGC0AAAAAABcBowUAAAAA4CJgtAAAAAAAXASMFgAAAACAi4DRAgAAAABwETBaAAAAAAAuAkYLAAAAAMBFwGgBAAAAALgIGC0AAAAAABcBowUAAAAA4CJgtAAAAAAAXASMFgAAAACAi4DRAgAAAABwETBaAAAAAAAuAkYLAAAAAMBFwGgBAAAAALgIGC0AAAAAABcBowUAAAAA4CJgtAAAAAAAXASMFgAAAACAi4DRAgAAAABwEf8FMTBfWSV2rYkAAAAASUVORK5CYII=>
```

Comment:
add screenshot?

Reply:
Better have it as separate file and include it?